aws/data_aws_unity_catalog_policy.go

```diff
@@ -15,6 +15,7 @@ import (
 func generateReadContext(ctx context.Context, d *schema.ResourceData, m *common.DatabricksClient) error {
 	bucket := d.Get("bucket_name").(string)
 	awsAccountId := d.Get("aws_account_id").(string)
+	awsPartition := d.Get("aws_partition").(string)
```

Comment:
we need to add validation for a value here as well...

Reply:
https://github.com/databricks/terraform-provider-databricks/blob/0c45e23268ddfd367934fea888f604828e6b2f23/aws/data_aws_unity_catalog_policy.go#L99 takes care of it or do we still need an extra validation here?

Reply:
ah, yes - missed it.